spec/integration/apo_spec.rb

```diff
@@ -6,20 +6,29 @@
   let(:user) { create(:user) }
   let(:new_apo_druid) { 'druid:zy987wv6543' }
   let(:new_collection_druid) { 'druid:zy333wv6543' }
+  let(:new_apo) { Dor::AdminPolicyObject.new(pid: new_apo_druid) }
+  let(:new_collection) { Dor::Collection.new(pid: new_collection_druid) }
 
   after do
     Dor::AdminPolicyObject.find(new_apo_druid).destroy # clean up after ourselves
     Dor::Collection.find(new_collection_druid).destroy # clean up after ourselves
   end
 
   before do
-    allow(Dor::SuriService).to receive(:mint_id).and_return(new_apo_druid, new_collection_druid)
+    allow_any_instance_of(ApoForm).to receive(:register_model).and_return(new_apo)
+    allow_any_instance_of(CollectionForm).to receive(:register_model).and_return(new_collection)
     allow(ApoController).to receive(:update_index).with(any_args)
     sign_in user, groups: ['sdr:administrator-role']
   end
 
   it 'creates and edits an apo' do
-    expect(Dor::Config.workflow.client).to receive(:create_workflow).twice
+    # TODO: This was removed when we switched from using
```

Comment:
I don't think this is going to improve the tests. Can we just remove this?

Reply:
I would be happy to ditch it. Coming soon.